packages/fast-components-styles-msft/src/call-to-action/index.ts

```diff
@@ -36,9 +33,10 @@ export const callToActionButtonOverrides: ComponentStyles<
             "& $button_contentRegion": {
                 transform: (config: DesignSystem): string => {
                     const designSystem: DesignSystem = withDesignSystemDefaults(config);
+                    const xTranslatePx: string = toPx(designSystem.designUnit);
```

Comment:
From a principled standpoint, is this always relative to the design unit, or is it mapped because they're the same number currently?

Reply:
Always. It's one unit of space from the text, similar to the two or three units we use for other horizontal spacing depending on density. I don't think this needs to adjust based on density though, even as the font size increases or decreases.